src/targets/nuget.ts

```diff
@@ -106,8 +110,19 @@ export class NugetTarget extends BaseTarget {
     await Promise.all(
       packageFiles.map(async (file: RemoteArtifact) => {
         const path = await this.artifactProvider.downloadArtifact(file);
+
+        // If an artifact containing a .snupkg file exists with the same base
+        // name as the .nupkg file, then download it to the same location.
+        // It will be picked up automatically when pushing the .nupkg.
+        const symbolFileName = file.filename.replace('.nupkg', '.snupkg');
```

Comment:
would be nice to add a test that these two extensions work and some third e.g `*.zip` doesn't

Reply:
Well, we don't have _any_ tests for nuget, (though [we do for some other targets](https://github.com/getsentry/craft/tree/master/src/targets/__tests__)) so we'd have to start there first I think.

As for picking up other files, that would be more of a test on `getArtifactsForRevision`.  All I'm doing here is matching `*.nupkg` with `*.snupkg` if it exists, and downloading it so it's in the same path before publishing.

`dotnet nuget push` automatically publishes a `.snupkg` if it is side-by-side the `.nupkg` in the same folder.

Reply:
Actually, nevermind.  I overcomplicated this.  Replacing with a simpler approach.